src/main/java/ProjektiPizza/demo/security/JwtAuthenticationTokenFilter.java

```diff
@@ -0,0 +1,36 @@
+package ProjektiPizza.demo.security;
+
+import ProjektiPizza.demo.token.JwtAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.AuthenticationException;
+import org.springframework.security.web.authentication.AbstractAuthenticationProcessingFilter;
+
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class JwtAuthenticationTokenFilter extends AbstractAuthenticationProcessingFilter {
+
+    public JwtAuthenticationTokenFilter() {
+        super("/api/**");
```

Comment:
Is it required authentication if I try to hit `/test-api/`?

Reply:
no it will't work.
it works only for request api/another request